.github/copilot-instructions.md

```diff
@@ -0,0 +1,139 @@
+# Copilot Instructions for Hydro
+
+## Project Overview
+
+Hydro is a high-level distributed programming framework for Rust. The repository is structured as a Cargo workspace monorepo containing multiple packages for distributed systems programming and dataflow processing.
+
+## Repository Structure
+
+- `dfir_rs` - Main DFIR package containing the DFIR runtime (scheduled layer)
+- `dfir_lang`, `dfir_macro` - Flow syntax compiler (compiled layer)
+- `hydro_lang`, `hydro_std`, `hydro_test` - Hydro functional syntax built on top of DFIR
+- `hydro_deploy/` - Framework for launching Hydro programs
+- `docs/` - Hydro.run website documentation
+- `website_playground/` - In-browser WASM playground for compiling DFIR
+- `benches/` - Microbenchmarks for DFIR and other frameworks
+- `design_docs/` - Architecture design documents
```

Comment:
@shadaj How would you describe the structure of `hydro_lang`, `hydro_std`, `hydro_test`, `hydro_deploy`? Those should be first above DFIR, lol